### DISCUSSION TITLE
A color palette approved by everyone.

### DISCUSSION THREAD

**Discussion by SebastJava** [SebastJava/mint-themes#15] [General]
I still think the official Mint-Y color palette is in serious need of a change. But my Mint-Yz color palette, while appreciated by a lot of people, might not be approved by everyone. So, it might be better, in the near future, to test a color palette made by a professional.

Examples:
https://material-ui.com/customization/color/#2014-material-design-color-palettes
https://flatuicolors.com/

I would start by choosing only one color palette, from one mind, one vision. But then i could include the two most popular colors from the old palette, just to help in getting general approval: the Mint green and the blue. And i would rename it MintClassic and BlueClassic. And i could also include two colors of mine, let's say my Orange and my Aqua. And i could rename it OrangeShine and AquaShine. And, finally, i could trim down the number of colors from the "professional" palette to the most popular 8 colors, so to make a palette that sums up to 12 colors.

In summary, in the hope of reaching a consensus, I would make a 12 color palette made up of 8 well balanced and professionally chosen colors, plus 2 “Classic” colors, and 2 “Shine” colors.

I should:
* Search the web for some flat or material design color palette made by professionals that could get general approval
* Create a custom mix of 8 new "professionally certified" colors plus 2 "Classic" and 2 "Shine" colors
* Get someone to approve and/or modify my currently waiting 5 PRs on https://github.com/linuxmint/mint-themes/pulls
* Create a new `constants.py` for Mint-Y
* Put this all together and create a complete, ready-to-use Debian package.

**Comment by SebastJava** [comment]
Done :
![Mint-Y-Variations-compare-OLD](https://user-images.githubusercontent.com/31940713/124156523-b8af9980-da65-11eb-9022-b221a5cfacb1.png)
**Latest**: https://github.com/SebastJava/mint-yz-theme (now on the **newflat** branch)
**Install**: https://github.com/SebastJava/mint-yz-theme/releases/tag/v2.6.1
